recipe/run_conda_forge_build_setup_win

```diff
@@ -8,5 +8,5 @@ conda install -n root --yes --quiet jinja2 anaconda-client
 
 # see https://github.com/conda-forge/staged-recipes/pull/750#issuecomment-225165530
 conda install -n root --yes --quiet conda-build=1.20.1
-
+if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
```

Comment:
Yeah, I realized that was unclear and added a link in a comment that appears to have shown up just before yours.